content/applications/finance/payment_acquirers/ogone.rst

```diff
@@ -5,84 +5,107 @@ Ogone
+
+Enter the same **SHA-IN** passphrase in both :guilabel:`Checks for e-Commerce & Alias Gateway` and
+:guilabel:`Checks for DirectLink and Batch (Automatic)`. You can leave the IP address field blank.
+
+Your **SHA-IN** and **SHA-OUT** passphrases should be different, and between 16 and 32 characters
+long. Make sure to use the same **SHA-IN** and **SHA-OUT** passphrases throughout the entire Ogone
+configuration, as Odoo only allows a single **SHA-IN** and single **SHA-OUT** passphrase.
+
+In order to retrieve the **SHA-OUT** key, log into your Ogone account, go to
+:menuselection:`Configuration --> Technical --> Information --> Transaction feedback --> All
+transaction submission modes`, and get or generate your **API Key** and **Client Key**. Be careful
+to copy your API key as you’ll not be allowed to get it later without generating a new one.
+
+When done, head to :menuselection:`Configuration --> Technical Information --> Transaction Feedback`
+and check the following options:
+
+- The :guilabel:`URL` fields for :guilabel:`HTTP redirection in the browser` can be left empty, as
+  Odoo will specify these URLs for every transaction request.
+- :guilabel:`I would like to receive transaction feedback parameters on the redirection URLs`:
+  should be checked.
+- :guilabel:`Direct HTTP server-to-server request`: should to be set to `Online but switch to a
+  deferred request when the online request fails`.
+- Both **URL** fields should contain the same following URL, with `<example>` replaced by your
+  database: `https://<example>/payment/ogone/return`.
+
+- :guilabel:`Dynamic eCommerce Parameters` should contain the following values: `ALIAS`, `AMOUNT`,
+  `CARDNO`, `CN`, `CURRENCY`, `IP`, `NCERROR` `ORDERID`, `PAYID`, `PM`, `STATUS`, `TRXDATE`. Other
+  parameters can be included (if you have another integration with Ogone that requires them), but
+  are not advised.
+- In the :guilabel:`All transaction submission modes` section, fill out **SHA-OUT** passphrase and
+  disable `HTTP request for status change`.
+
+To allow your customers to save their credit card credentials for future use, head to
+:menuselection:`Configuration --> Alias --> My alias information`. From this tab, you can configure
+how the user can have its card details saved, for how long the information is saved, if a checkbox
+to save the card information should be displayed, etc.
 
 .. important::
-   If you are trying Ogone as a test, with the Test Account, change the **State** to *Test Mode*. We
-   recommend doing this on a test Odoo database, rather than on your main database.
-
-Configuration on Ogone
-======================
+   If you wish to run tests with Ogone, within the accounting app, go to
+   :menuselection:`Configuration --> Payment Acquirers`. Click on :guilabel:`Ogone`, enable
+   :guilabel:`Test Mode` and enter your credentials in the :guilabel:`Credentials` tab. We recommend
+   doing this on an Odoo test database, rather than on your main database.
```

Comment:
Maybe move this to Settings in Odoo since this is about the settings on the Odoo side? :)